include/unifex/stop_immediately.hpp

```diff
@@ -296,6 +296,10 @@ struct _stream<SourceStream, Values...>::type {
     operation<Receiver> connect(Receiver&& receiver) && {
       return operation<Receiver>{stream_, (Receiver&&)receiver};
     }
+    template<typename Receiver>
+    void connect(Receiver&& receiver) & =delete;
```

Comment:
Is this overload declaration necessary given that we have the `const &` overload below?

Reply:
I wanted to be explicit that this was removed on purpose. could be a comment..